MediaGalleryUi/Model/AssetDetailsProvider/CreatedAt.php

```diff
@@ -22,6 +21,20 @@ class CreatedAt implements AssetDetailsProviderInterface
      */
     private const DATE_FORMAT = 'd/m/Y, g:i A';
 
+    /**
+     * @var DateTime
+     */
+    private $date;
+
+    /**
+     * @param DateTime $date
+     */
+    public function __construct(
+        DateTime $date
```

Comment:
I think it's better to use `\Magento\Framework\Stdlib\DateTime\TimezoneInterface` for retrieving and formatting a date (only for view layer, not applicable to `SynchronizeFiles` class)